src/lib/notcurses.c

```diff
@@ -532,6 +532,30 @@ make_ncpile(notcurses* nc, ncplane* n){
   return ret;
 }
 
+
+static inline size_t  ncplane_sizeof_cellarray( unsigned rows, unsigned cols)
+{
+  size_t fbsize;
+  size_t size;
+
+  // (nat) This protects against size_t overflow and also checks
+  // that dimensions are not zero en-passant. Why ?
```

Comment:
while i celebrate erudition in all its forms, i have never seen a hyphen used in "en passant", and it certainly isn't supported by the source french

Reply:
That's just artistic license :wink: